src/dump.c

```diff
@@ -1916,7 +1930,11 @@ static void jl_save_system_image_to_stream(ios_t *f)
     jl_serialize_value(f, jl_typeinf_func);
     jl_serialize_value(f, jl_type_type->name->mt);
 
-    intptr_t i=2;
```

Comment:
need to restore the code to scan generated code for references to these (https://github.com/JuliaLang/julia/commit/735316ec6158ddd0716e3a3f839eb3dd343058e7#diff-669d4cc5c9c8f4573c5f8d57f5dcab20L336) before erasing them